### PR TITLE
fix: make nouns singular

### PR DESCRIPTION
**Pull request by adriantam**
## Description
Update CLI so that noun is singular

```
OpenFGA CLI

Usage:
  fga [command]

Available Commands:
  completion  Generate the autocompletion script for the specified shell
  help        Help about any command
  model       Write, Read and List authorization models in a store
  query       Query a store.
  store       Create, Get, Delete and List OpenFGA Stores
  tuple       Interact with Relationship Tuples in a store.

Flags:
      --api-audience string       API Audience. Used when performing the Client Credentials flow
      --api-token string          API Token. Will be sent in as a Bearer in the Authorization header
      --api-token-issuer string   API Token Issuer. API responsible for issuing the API Token. Used in the Client Credentials flow
      --client-id string          Client ID. Sent to the Token Issuer during the Client Credentials flow
      --client-secret string      Client Secret. Sent to the Token Issuer during the Client Credentials flow
      --config string             config file (default is $HOME/.fga.yaml)
  -h, --help                      help for fga
      --server-url string         OpenFGA API URI e.g. https://api.fga.example:8080 (default "http://localhost:8080")

Use "fga [command] --help" for more information about a command.
```

## References
Close https://github.com/openfga/cli/issues/50

## Review Checklist
- [x] I have clicked on ["allow edits by maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [ ] I have added documentation for new/changed functionality in this PR or in a PR to [openfga.dev](https://github.com/openfga/openfga.dev) [Provide a link to any relevant PRs in the references section above]
- [x] The correct base branch is being used, if not `main`
- [ ] I have added tests to validate that the change in functionality is working as expected

### CODE REVIEW

**Review by rhamzeh — CHANGES_REQUESTED** [review]
Not all nouns should be, just the nouns in the command itself (`fga {noun} ...`):
- stores -> store
- tuples -> tuple
- models -> model

For the README, the headers should remain the same to be consistent with the SDKs

StoresCmd, ModelsCmd should be StoreCmd, ModelCmd

File: README.md [hunk 1]
```diff
@@ -17,17 +17,17 @@ A cross-platform CLI to interact with an OpenFGA server
 - [Usage](#usage)
   - [Configuration](#configuration)
   - [Commands](#commands)
-    - [Stores](#stores)
```
Comment (rhamzeh) [inline README.md:20]: Let's not change these headers

File: cmd/model/model.go [hunk 2]
```diff
@@ -24,7 +24,7 @@ import (
 
 // ModelsCmd represents the store command.
 var ModelsCmd = &cobra.Command{
```
[the remark this answers is not in the document]
Comment (rhamzeh) [reply cmd/model/model.go:26]: Same in the comment and in the stores

File: README.md (hunk 1 above)
Comment (adriantam) [reply README.md]: updated

File: cmd/model/model.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (adriantam) [reply cmd/model/model.go]: updated

**Review by rhamzeh — APPROVED** [review]

**Pull request merged**